pom.xml

```diff
@@ -32,6 +32,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>11</java.version>
+<!--    <os.detected.classifier>osx-x86_64</os.detected.classifier>-->
```

Comment:
this line was used to build on my M1 Mac, can be removed from the code going to be merged to main

Reply:
I actually liked leaving it, I think it is useful to have around for others building with an M1 (self included) - thoughts?